src/app/domain/ReleaseInfo.php

```diff
@@ -54,6 +54,14 @@ public function isUnsafeVersion(): bool
   {
     return file_exists($this->getPath() . '/unsafe.version');
   }
+  
+  public function getUnsafeReasons(): array
+  {
+    $unsaveFile = $this->getPath() . '/unsafe.version';
```

Comment:
line 55 and line 60 has some duplicated code. can you move it to an own method?

private function getUnsafeVersionPath(): string {
...
}